prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -26,15 +32,27 @@ def get_cached_discover_method(ttl: int) -> Callable:
         """
 
         @ttl_cache(ttl=ttl)
-        def _discover() -> bool:
-            """Return true if redfish services have been discovered."""
-            logger.info("Discovering redfish services...")
-            services = redfish.discover_ssdp()
-            if len(services) == 0:
-                logger.info("No redfish services discovered")
-                return False
-            logger.debug("Discovered redfish services: %s", services)
-            return True
+        def _discover(host: str, username: str, password: str) -> bool:
```

Comment:
Based on your code here. we don't need username/password.
https://github.com/canonical/hardware-observer-operator/pull/69/files

Reply:
You're right, I missed that. Thanks for pointing it out. I've made the required changes in the latest commit.